### PR TITLE
Add dataprep common service

### PR DESCRIPTION
**Pull request by yongfengdu**
Add dataprep component

## Description

The summary of the proposed changes as long as the relevant motivation and context.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds new functionality)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Review by daisy-ycguo — APPROVED** [review]
lgtm

File: helm-charts/chatqna/gaudi-values.yaml [hunk 1]
```diff
@@ -9,25 +9,20 @@ replicaCount: 1
...
 
-securityContext:
-  allowPrivilegeEscalation: false
-  runAsNonRoot: true
-  runAsUser: 1000
-  capabilities:
-    drop:
-    - ALL
-  seccompProfile:
-    type: RuntimeDefault
```
Comment (lianhao) [inline helm-charts/chatqna/gaudi-values.yaml:29]: why remove default securityContext here?

File: helm-charts/chatqna/values.yaml [hunk 2]
```diff
@@ -10,24 +10,19 @@ replicaCount: 1
...
-securityContext:
-  allowPrivilegeEscalation: false
-  runAsNonRoot: true
-  runAsUser: 1000
-  capabilities:
-    drop:
-    - ALL
-  seccompProfile:
-    type: RuntimeDefault
-
```
Comment (lianhao) [inline helm-charts/chatqna/values.yaml:30]: ditto?

**Review by lianhao — CHANGES_REQUESTED** [review]
Why remove the default security context in value files?

File: helm-charts/chatqna/gaudi-values.yaml (hunk 1 above)
Comment (yongfengdu) [reply helm-charts/chatqna/gaudi-values.yaml:29]: I was working with an old version and these values was overwritten(Include the volume name change).
Will restore them in next revision.

**Review by lianhao — APPROVED** [review]

**Pull request merged**